dm/dumpling/dumpling.go

```diff
@@ -196,7 +201,43 @@ func (m *Dumpling) Update(context.Context, *config.SubTaskConfig) error {
 // Status implements Unit.Status.
 func (m *Dumpling) Status(_ *binlog.SourceStatus) interface{} {
 	// NOTE: try to add some status, like dumped file count
-	return &pb.DumpStatus{}
+	mid := &export.Midparams{}
+	mid = m.realdumper.GetParameters()
+	s := &pb.DumpStatus{
+		TotalTables:       mid.TotalTables,
+		CompletedTables:   mid.CompletedTables,
+		FinishedBytes:     mid.FinishedBytes,
+		FinishedRows:      mid.FinishedRows,
+		EstimateTotalRows: mid.EstimateTotalRows,
+	}
+	go m.printStatus()
+	return s
+}
+
+const logProgressTick = 2 * time.Minute
+
+func (m *Dumpling) printStatus() {
```

Comment:
this function is added for other units to peroidically print status to log, but dumpling itself will do this so we don't bother write `printStatus` in DM

Reply:
emm, I understand that I should remove function `printStatus()`, but how dumpling print status by itself?

Reply:
https://github.com/pingcap/tidb/blob/cbe5240dd0367d8a6ae882e869dd3a340fb15e77/dumpling/export/dump.go#L223

And dumpling use the same logger with DM, so logs are written to same file.